ingestion/Dockerfile.ci

```diff
@@ -32,6 +32,18 @@ RUN apt-get update \
     && ACCEPT_EULA=Y apt-get install -y msodbcsql18 \
     && rm -rf /var/lib/apt/lists/*
 
+# Oracle instant client for thick mode
+# https://python-oracledb.readthedocs.io/en/latest/user_guide/initialization.html#enablingthick
+RUN apt-get update; \
+  apt-get install libaio1 alien unzip -y;
```

Comment:
Instead of separately running another Docker `RUN` Instruction, let's include these in the already available `RUN` instruction which installs apt dependencies!

![image](https://user-images.githubusercontent.com/15995028/213214154-ccfe5a6b-b308-401b-af95-d2db3b1a04fe.png)